src/fragments/lib/in-app-messaging/js/getting-started.mdx

```diff
@@ -0,0 +1,115 @@
+Make sure you have completed the below steps:
+
+- [Completed the pre-requisite steps.](/lib/in-app-messaging/prerequisites)
+- [Create a campaign from the pinpoint console.](/lib/in-app-messaging/create-campaigns)
+
+## Integrate your application
+
+In your application directory, you should first Install necessary dependencies.Although Amplify In-App Messaging can be used as a standalone JavaScript library, the only UI integration currently available is only for _React Native_. The integrations steps here assume a React Native application as the implementation target.
+
+```bash
+npm install aws-amplify@in-app-messaging aws-amplify-react-native@in-app-messaging amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker
+```
+
+You will also need to install pod dependencies for `iOS`. Navigate the the ios directory in your project and run the install command.
+
+```bash
+pod install
+```
+The AWS SDK requires React Native applications to polyfill `crypto.getRandomValues` and `URL`. Import it at the top of your application's root entry point. For example, `index.js`.
+```js
+import 'react-native-get-random-values';
+import 'react-native-url-polyfill/auto';
+Next, you should import and load the `aws-exports.js` configuration file created and modified in an earlier step into your application. Add the Amplify configuration step to your application’s root entry point. For example, `index.js`.
+
+```js
+import Amplify from 'aws-amplify';
+import awsconfig from './src/aws-exports';
+
+Amplify.configure(awsconfig);
+```
+
+Finally, integrate the Amplify React Native UI components for In-App Messaging in your application’s root component. For example, `App.jsx`.
+
+```jsx
+import {
+  InAppMessagingProvider,
+  InAppMessageDisplay
+} from 'aws-amplify-react-native';
+
+const App = () => (
+  <InAppMessagingProvider>
+    {/* Your application */}
+    <InAppMessageDisplay />
+  </InAppMessagingProvider>
+);
+```
+
+Now your application is set up with Amplify In-App Messaging. To interact with Amplify In-App Messaging APIs, you will first need to import the `Notifications` category.
+
+```js
+import { Notifications } from 'aws-amplify';
+```
+
+The In-App Messaging feature is a subcategory of Notifications. To make it easier to access throughout your application, you can de-structure it.
+
+```js
+const { InAppMessaging } = Notifications;
+```
+
+Below is an example of what your `App.tsx` file should look like:
```

Comment:
Should we be assuming that users Typescript? Think for getting started it may make more sense to use vanilla JS